test/src/KeyResetMessageTest.cc

```diff
@@ -549,92 +549,6 @@ TEST_F(KeyResetMessageTest, check_receive_message_to_revoked_key_from_unknown) {
-    outgoing_msg->shortmsg = strdup("Well isn't THIS a useless message...");
-    outgoing_msg->longmsg = strdup("Hi Mom...\n");
-    outgoing_msg->attachments = new_bloblist(NULL, 0, "application/octet-stream", NULL);
-    output_stream << "Message created.\n\n";
-    output_stream << "Encrypting message as MIME multipart…\n";
-    message* enc_outgoing_msg = nullptr;
-    output_stream << "Calling encrypt_message()\n";
-    status = encrypt_message(session, outgoing_msg, NULL, &enc_outgoing_msg, PEP_enc_PGP_MIME, 0);
-    ASSERT_EQ(status , PEP_UNENCRYPTED);
-    //
-    output_stream << "Message created." << endl;
-
-    // Make the update have occurred earlier, so we don't notify her
-    // (We have no key for her yet anyway!)
-    int int_result = sqlite3_exec(
-        session->db,
-        "update identity "
-        "   set timestamp = '2018-04-10 16:48:33' "
-        "   where address = 'pep-test-gabrielle@pep-project.org' ;",
-        NULL,
-        NULL,
-        NULL
-    );
-    ASSERT_EQ(int_result , SQLITE_OK);
-
-    // FIXME: longer term we need to fix the test, but the key attached to the message below has expired, so for now, we give her a new key
-    slurp_and_import_key(session, "test_keys/pub/pep-test-gabrielle-0xE203586C_pub.asc");
-
-    status = key_reset(session, alice_fpr, from_ident);
-    ASSERT_OK;
-    ASSERT_EQ(m_queue.size() , 0);
-    m_queue.clear();
-
-    // Now we get mail from Gabi, who only has our old key AND has become
-    // a pEp user in the meantime...
-    message* received_mail = slurp_message_file_into_struct("test_files/398_gabrielle_to_alice.eml");
-    message* decrypted_msg = NULL;
-    stringlist_t* keylist = NULL;
-    PEP_decrypt_flags_t flags = 0;
-
-    // We expect the app to provide user_ids wherever it has them on incoming messages, and since it breaks things
-    // here if we don't put it there, we do now.
-    received_mail->to->ident->user_id = strdup(PEP_OWN_USERID);
-
-    status = decrypt_message_2(session, received_mail,
-                                  &decrypted_msg, &keylist, &flags);
-
-    ASSERT_EQ(m_queue.size() , 1);
-    vector<message*>::iterator it = m_queue.begin();
-    message* reset_msg = *it;
-    ASSERT_NOTNULL(reset_msg);
-    ASSERT_NOTNULL(reset_msg->from);
-    ASSERT_NOTNULL(reset_msg->to);
-    ASSERT_NOTNULL(reset_msg->to->ident);
-    ASSERT_STREQ(reset_msg->to->ident->address, "pep-test-gabrielle@pep-project.org");
-    ASSERT_STREQ(reset_msg->to->ident->fpr, "906C9B8349954E82C5623C3C8C541BD4E203586C");
-    ASSERT_STRNE(reset_msg->from->fpr, alice_fpr);
-    ASSERT_NOTNULL(keylist);
-    ASSERT_NOTNULL(keylist->value);
-    ASSERT_STRNE(keylist->value, alice_fpr);
-    ASSERT_NOTNULL(keylist->next);
-    if (strcmp(keylist->next->value, "906C9B8349954E82C5623C3C8C541BD4E203586C") != 0) {
-        ASSERT_NOTNULL(keylist->next->next);
-        ASSERT_STREQ(keylist->next->value, "906C9B8349954E82C5623C3C8C541BD4E203586C");
-    }
-}
-
-
```

Comment:
Is this test replaced anywhere?

Reply:
(I assume it is on the other test files and this was left as redundant mostly a confirmation)